deepchem/data/data_loader.py

```diff
@@ -18,7 +18,7 @@
 from deepchem.utils.save import log
 from deepchem.utils.save import load_csv_files
 from deepchem.utils.save import load_sdf_files
-from deepchem.utils.save import encode_fasta_sequence
```

Comment:
Can you mark the code in deepchem.util.save as deprecated and forward the calls to deepchem.utils.genomics?

Reply:
Yes I'll do that, one thing though I plan to remove _seq_to_encoded from the save.py completely because functions starting with _ can be removed without notice and people should not generally use it in their code? Sounds good?

Reply:
Sounds good to me.

Reply:
Are the messages okay? Is this what you expected? or should I do anything more?

Reply:
These messages are sufficient.